app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/DatasetsOverviewActivityTest.kt

```diff
@@ -42,7 +42,7 @@ class DatasetsOverviewActivityTest {
     @get:Rule
     var hiltRule = HiltAndroidRule(this)
     @get:Rule
-    val activityScenarioRule: ActivityScenarioRule<AddPictureActivity> = ActivityScenarioRule(
+    val activityScenarioRule: ActivityScenarioRule<TakePictureActivity> = ActivityScenarioRule(
         Intent(
             ApplicationProvider.getApplicationContext(),
             DatasetsOverviewActivity::class.java
```

Comment:
It is probably my bad from previous PRs but I think the activityScenarioRule should be ActivityScenarioRule<DatasetsOverviewActivity> no? Or am I missing something?

Reply:
that's an old mistake but you're correct, it's the wrong type. I'll fix that now.